asaps/models.py

```diff
@@ -66,20 +66,23 @@ def download_json(rec_obj):
     f = open(file_name + '.json', 'w')
     json.dump(rec_obj, f)
     f.close()
+    return f.name
 
 
-def createcsv(csv_data, file_name):
+def create_csv(csv_data, file_name):
     """Create CSV file from list of dicts.
 
     Example: {'uri': rec_obj.uri,
     'old_value': old_value , 'new_value': new_value}.
     """
     date = datetime.datetime.now().strftime('%Y-%m-%d %H.%M.%S')
     header = list(csv_data[0].keys())
-    f = csv.DictWriter(open(file_name + date + '.csv', 'w'), fieldnames=header)
+    full_file_name = file_name + date + '.csv'
+    f = csv.DictWriter(open(full_file_name, 'w'), fieldnames=header)
```

Comment:
It's good practice to make sure you are closing a file when you're done with it. The easiest way to do this is to use a context manager which will close it for you even if an exception gets raised:
```python
with open(full_file_name, "w") as fp:
    f = csv.DictWriter(fp, fieldnames=header)
    ...
return full_file_name
```